docs/services/slam/cartographer/_index.md

```diff
@@ -54,15 +40,18 @@ The `cartographer` module supports three modes of operation:
 Creating and updating SLAM maps with Cartographer is especially CPU-intensive, so the `cartographer` modular resource runs in the cloud for these two tasks.
 For doing pure localization on an existing map, the `cartographer` modular resource runs locally on your machine.
```

Comment:
Can remove, is redundant with the "Running cartographer in the cloud" note below